hosts/hetzner-ephemeral-alpha/default.nix

```diff
@@ -0,0 +1,69 @@
+{ pkgs, config, inputs, lib, ... }:
+{
+  # User options
+  users = {
+    juuso.authorizedKeys = [ "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBNMKgTTpGSvPG4p8pRUWg1kqnP9zPKybTHQ0+Q/noY5+M6uOxkLy7FqUIEFUT9ZS/fflLlC/AlJsFBU212UzobA= ssh@secretive.sandbox.local" ];
+    kari.authorizedKeys = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIKEdpdbTOz0h9tVvkn13k1e8X7MnctH3zHRFmYWTbz9T kari@torque" ];
+  };
+
+  # Users group
+  users.groups.users = { };
+
+  # Localization
+  networking.hostName = "hetzner-ephemeral-alpha";
+  time.timeZone = "Europe/Helsinki";
+
+  # Automatic carbage collection
+  nix.gc = {
+    automatic = true;
+    dates = "weekly";
+    options = "--delete-older-than 30d";
+  };
```

Comment:
We do not want this. A better solution is a systemd script with a disk size filter which will clear X amount of caches. For now, lets just delete this.